CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Breaking changes:
 - [#1574](https://github.com/rails-api/active_model_serializers/pull/1574) Default key case for the JsonApi adapter changed to dashed. (@remear)
 
 Features:
+- [#1700](https://github.com/rails-api/active_model_serializers/pull/1700) Support pagination link for Kaminari when no data is returned
```

Comment:
oh, super sorry, there should be headers above this in the changelog for 'master'

```plain
### [master](https://github.com/rails-api/active_model_serializers/compare/v0.10.0.rc5...master)

Breaking changes:

Features:
- [#1700](https://github.com/rails-api/active_model_serializers/pull/1700) Support pagination link for Kaminari when no data is returned


Fixes:

Misc:
```